attended-events/part_1_data_engineering.ipynb

```diff
@@ -762,6 +762,8 @@
    "outputs": [],
    "source": [
     "# Selecting the target fields\n",
+    "# Please bear in mind that if running this cell gave you an exception, you are probably using an outdated version of `predicthq` library.\n",
+    "# Make sure to install requirements.txt if your running offline or the required libs if you're running on Colab, as advised above. Then run this notebook again.\n",
```

Comment:
```suggestion
    "# If running this cell gave you an exception, make sure you're using the `predicthq` library version specified in requirements.txt\n",
    "# or the version advised above if you're running this notebook on Colab. Then run this notebook again.\n",
```

And similarly for the other blocks. There's a typo in the original text ("your" instead of "you're"); I also _tried_ to make the message less verbose.